### DISCUSSION TITLE
connection between EC2 AWS istance and zyxel

### DISCUSSION THREAD

**Discussion by alberto-fraccarollo** [strongswan/strongswan#413] [Q&A]
I start with command ipsec start
when I see the 'Network is unreachable' logs
May 21 15:57:24 08 [IKE] <fa-to-aviva | 1> initiating IKE_SA fa-to-aviva [1] to 82.62.227.24
May 21 15:57:24 08 [ENC] <fa-to-aviva | 1> generating IKE_SA_INIT request 0 [SA KE No N (NATD_S_IP) N (NATD_D_IP) N (FRAG_SUP) N (HASH_ALG) N (REDIR_SUP)]
May 21 15:57:24 08 [NET] <fa-to-aviva | 1> sending packet: from 129.114.108.9 [500] to 82.62.227.24 [500] (336 bytes)
May 21 15:57:24 03 [NET] error writing to socket: Network is unreachable
May 21 15:57:28 04 [IKE] <fa-to-aviva | 1> retransmit 1 of request with message ID 0
May 21 15:57:28 04 [NET] <fa-to-aviva | 1> sending packet: from 129.114.108.9 [500] to 82.62.227.24 [500] (336 bytes)
May 21 15:57:28 03 [NET] error writing to socket: Network is unreachable
May 21 15:57:35 11 [IKE] <fa-to-aviva | 1> retransmit 2 of request with message ID 0
May 21 15:57:35 11 [NET] <fa-to-aviva | 1> sending packet: from 129.114.108.9 [500] to 82.62.227.24 [500] (336 bytes)
May 21 15:57:35 03 [NET] error writing to socket: Network is unreachable
May 21 15:57:48 12 [IKE] <fa-to-aviva | 1> retransmit 3 of request with message ID 0
May 21 15:57:48 12 [NET] <fa-to-aviva | 1> sending packet: from 129.114.108.9 [500] to 82.62.227.24 [500] (336 bytes)
May 21 15:57:48 03 [NET] error writing to socket: Network is unreachable
May 21 15:58:12 13 [IKE] <fa-to-aviva | 1> retransmit 4 of request with message ID 0
May 21 15:58:12 13 [NET] <fa-to-aviva | 1> sending packet: from 129.114.108.9 [500] to 82.62.227.24 [500] (336 bytes)
May 21 15:58:12 03 [NET] error writing to socket: Network is unreachable

below my ipsec.config

# ipsec.conf - strongSwan IPsec configuration file
config setup
        charondebug="all"
        uniqueids=yes
conn fa-to-aviva
        type=tunnel
        auto=start
        keyexchange=ikev2
        authby=secret
        left=129.114.108.9
        leftsubnet=10.52.0.0/22
        right=82.62.227.24
        rightsubnet=192.168.24.0/24
        ike=aes256-sha1-modp1024!
        esp=aes256-sha1!
        aggressive=no
        keyingtries=%forever
        ikelifetime=28800s
        lifetime=3600s
        dpddelay=30s
        dpdtimeout=120s
        dpdaction=restart

How could fix it?
regards

A

**Comment by Thermi** [comment]
> 129.114.108.9

That's probably not a local IP. You can only send packets from a local IP.

**Reply by alberto-fraccarollo** [reply]
Hi, 
thanks for your reply 

this is my actual config

# ipsec.conf - strongSwan IPsec configuration file
config setup
        charondebug="all"
        uniqueids=yes
        strictcrlpolicy=no

# connection to eleva AWS
conn moma-to-eleva
  authby=secret
  left=%defaultroute
  leftid=public IP
  leftsubnet=10.52.0.0/22
  right=public IP
  rightsubnet=172.31.16.0/20
  ike=aes256-sha2_256-modp1024!
  esp=aes256-sha2_256!
  keyingtries=0
  ikelifetime=1h
  lifetime=8h
  dpddelay=30
  dpdtimeout=120
  dpdaction=restart
  auto=start

this il my setting firewall

moma
sudo iptables -t nat -A POSTROUTING -s 172.31.16.0/20 -d 10.52.0.0/22 -j MASQUERADE
 
eleva
sudo iptables -t nat -A POSTROUTING -s 10.52.0.0/22 -d 172.31.16.0/20 -j MASQUERADE

I recive this message error 

May 24 13:49:23 08[ENC] <moma-to-eleva|1> generating IKE_SA_INIT request 0 [ SA KE No N(NATD_S_IP) N(NATD_D_IP) N(FRAG_SUP) N(HASH_ALG) N(REDIR_SUP) ]
May 24 13:49:23 08[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[500] to 52.215.8.13[500] (336 bytes)
May 24 13:49:27 09[IKE] <moma-to-eleva|1> retransmit 1 of request with message ID 0
May 24 13:49:27 09[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[500] to 52.215.8.13[500] (336 bytes)
May 24 13:49:34 11[IKE] <moma-to-eleva|1> retransmit 2 of request with message ID 0
May 24 13:49:34 11[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[500] to 52.215.8.13[500] (336 bytes)
May 24 13:49:47 12[IKE] <moma-to-eleva|1> retransmit 3 of request with message ID 0
May 24 13:49:47 12[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[500] to 52.215.8.13[500] (336 bytes)
May 24 13:50:11 13[IKE] <moma-to-eleva|1> retransmit 4 of request with message ID 0
May 24 13:50:11 13[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[500] to 52.215.8.13[500] (336 bytes)
May 24 13:50:53 14[IKE] <moma-to-eleva|1> retransmit 5 of request with message ID 0
May 24 13:50:53 14[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[500] to 52.215.8.13[500] (336 bytes)

regards 

A

**Reply by Thermi** [reply]
The other peer does not reply. Verify firewall rules, specifically AWS VPC SGs, NACLS, and iptables. Also verify routes in the VPCs.

**Reply by alberto-fraccarollo** [reply]
Hi, 
thanks for your replay. Now the tunnel connection is stable but I don't ping my leftid=public IP
How could fix it?

regards

A

**Reply by Thermi** [reply]
Hi

Please provide the output of `swanctl -l` and your ping command.

**Reply by alberto-fraccarollo** [reply]
Hi, 
thanks for your replay

now I recive correctly package from  2.45.7.13 
but ping non responding
How could fix it?

A.


Jun  7 10:11:35 08[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:11:45 09[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:11:45 09[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 24 [ ]
Jun  7 10:11:45 09[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:11:45 10[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:11:45 10[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 24 [ ]
Jun  7 10:11:46 04[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:11:46 04[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 27 [ ]
Jun  7 10:11:46 04[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 27 [ ]
Jun  7 10:11:46 04[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:06 14[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:12:16 13[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:12:16 13[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 25 [ ]
Jun  7 10:12:16 13[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:16 05[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:16 05[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 25 [ ]
Jun  7 10:12:17 06[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:17 06[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 28 [ ]
Jun  7 10:12:17 06[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 28 [ ]
Jun  7 10:12:17 06[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:37 09[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:12:47 04[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:12:47 04[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 26 [ ]
Jun  7 10:12:47 04[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:47 11[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:47 11[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 29 [ ]
Jun  7 10:12:47 11[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 29 [ ]
Jun  7 10:12:47 11[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:47 12[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:47 12[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 26 [ ]
Jun  7 10:11:35 08[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:11:45 09[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:11:45 09[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 24 [ ]
Jun  7 10:11:45 09[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:11:45 10[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:11:45 10[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 24 [ ]
Jun  7 10:11:46 04[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:11:46 04[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 27 [ ]
Jun  7 10:11:46 04[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 27 [ ]
Jun  7 10:11:46 04[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:06 14[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:12:16 13[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:12:16 13[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 25 [ ]
Jun  7 10:12:16 13[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:16 05[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:16 05[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 25 [ ]
Jun  7 10:12:17 06[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:17 06[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 28 [ ]
Jun  7 10:12:17 06[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 28 [ ]
Jun  7 10:12:17 06[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:37 09[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:12:47 04[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:12:47 04[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 26 [ ]
Jun  7 10:12:47 04[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:47 11[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:47 11[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 29 [ ]
Jun  7 10:12:47 11[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 29 [ ]
Jun  7 10:12:47 11[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:12:47 12[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:12:47 12[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 26 [ ]
Jun  7 10:13:07 13[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:13:17 05[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:13:17 05[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 27 [ ]
Jun  7 10:13:17 05[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:13:17 06[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:13:17 06[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 27 [ ]
Jun  7 10:13:18 08[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:13:18 08[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 30 [ ]
Jun  7 10:13:18 08[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 30 [ ]
Jun  7 10:13:18 08[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:13:38 10[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:13:48 11[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:13:48 11[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 28 [ ]
Jun  7 10:13:48 11[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:13:48 12[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:13:48 12[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 28 [ ]
Jun  7 10:13:49 14[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:13:49 14[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 31 [ ]
Jun  7 10:13:49 14[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 31 [ ]
Jun  7 10:13:49 14[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:14:09 05[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:14:19 08[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:14:19 08[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 29 [ ]
Jun  7 10:14:19 08[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:14:19 07[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:14:19 07[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 29 [ ]
Jun  7 10:14:20 09[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:14:20 09[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 32 [ ]
Jun  7 10:14:20 09[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 32 [ ]
Jun  7 10:14:20 09[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:14:40 11[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:14:50 14[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:14:50 14[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 30 [ ]
Jun  7 10:14:50 14[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:14:50 15[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:14:50 15[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 30 [ ]
Jun  7 10:14:51 13[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:14:51 13[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 33 [ ]
Jun  7 10:14:51 13[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 33 [ ]
Jun  7 10:14:51 13[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:15:11 08[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:15:21 09[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:15:21 09[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 31 [ ]
Jun  7 10:15:21 09[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:15:21 10[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:15:21 10[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 34 [ ]
Jun  7 10:15:21 10[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 34 [ ]
Jun  7 10:15:21 10[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:15:21 04[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:15:21 04[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 31 [ ]
Jun  7 10:15:41 14[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:15:51 15[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:15:51 15[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 32 [ ]
Jun  7 10:15:51 15[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:15:51 13[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:15:51 13[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 32 [ ]
Jun  7 10:15:52 05[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:15:52 05[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 35 [ ]
Jun  7 10:15:52 05[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 35 [ ]
Jun  7 10:15:52 05[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:16:12 07[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:16:22 10[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:16:22 10[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 33 [ ]
Jun  7 10:16:22 10[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:16:22 04[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:16:22 04[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 33 [ ]
Jun  7 10:16:23 11[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:16:23 11[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 36 [ ]
Jun  7 10:16:23 11[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 36 [ ]
Jun  7 10:16:23 11[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:16:43 15[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
Jun  7 10:16:53 05[IKE] <moma-to-eleva|1> sending DPD request
Jun  7 10:16:53 05[ENC] <moma-to-eleva|1> generating INFORMATIONAL request 34 [ ]
Jun  7 10:16:53 05[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:16:53 06[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:16:53 06[ENC] <moma-to-eleva|1> parsed INFORMATIONAL response 34 [ ]
Jun  7 10:16:54 08[NET] <moma-to-eleva|1> received packet: from 2.45.7.13[4500] to 10.52.3.89[4500] (80 bytes)
Jun  7 10:16:54 08[ENC] <moma-to-eleva|1> parsed INFORMATIONAL request 37 [ ]
Jun  7 10:16:54 08[ENC] <moma-to-eleva|1> generating INFORMATIONAL response 37 [ ]
Jun  7 10:16:54 08[NET] <moma-to-eleva|1> sending packet: from 10.52.3.89[4500] to 2.45.7.13[4500] (80 bytes)
Jun  7 10:17:14 10[IKE] <moma-to-eleva|1> sending keep alive to 2.45.7.13[4500]
